backend/src/main.py

```diff
@@ -34,16 +39,7 @@ async def lifespan(app: FastAPI):  # noqa: ARG001
 app = FastAPI(lifespan=lifespan)
 
 
-# Dependency to get the database session
-def get_db():
-    db = SessionLocal()
-    try:
-        yield db
-    finally:
-        db.close()
-
-
-@app.post("/users/", response_model=User)
+@app.post("/users/", response_model=UserRead)
```

Comment:
Let's remove trailing /. FastAPI handles redirections better that way, in my experience.